src/components/calculator/ExplanationCard/ExplanationCard.tsx

```diff
@@ -42,6 +45,43 @@ const calculationStepHeader = (header: string, value: string): JSX.Element => {
   )
 }
 
+function numberToReadable(value: number): string {
```

Comment:
This function is pretty hacky but I don't have a better idea - thoughts?

Reply:
It might be a little nicer if num2fraction had a helper function that returned a numerator and denominator rather than splitting here?